README.md

```diff
@@ -53,8 +53,28 @@ You can shut down the notebooks and close docker by entering control-c in the te
 
 ## Updates
 
-You can update your notebooks using the command:
+You can update the toolbox using the command:
 
 ```
 $ git pull
 ```
+
+After updating, run
+```
+bash start_docker.sh --update
+```
+or
+```
+bash start_docker.sh -u
+```
+
+to overwrite the old notebooks with the new updated versions.
+
+### WARNING
+
+If you have made changes to scripts that you would like to keep, rename them prior to running
+```
+bash start_docker.sh --update
```

Comment:
I think recopying the code here is confusing and makes them think they need to run it again. I would change this to something like: "prior to running the above command"